config/block_configuration.go

```diff
@@ -155,15 +152,9 @@ func (b *BlockConfiguration) ToBlock(orgName string, stackId int64) types.Block
 		IsShared:            b.IsShared,
 		DnsName:             "",
 		ModuleSource:        b.ModuleSource,
-		ModuleSourceVersion: b.ModuleSourceVersion,
+		ModuleSourceVersion: b.ModuleConstraint,
 		Connections:         b.Connections.Targets(),
 	}
-	for k, conn := range block.Connections {
```

Comment:
Is this removed because you aren't setting the stackId on the connections anymore?